machine/lenovo/lenovo_ne2572/machine.make

```diff
@@ -0,0 +1,51 @@
+# Lenovo NE2572
+
+ONIE_ARCH ?= x86_64
+SWITCH_ASIC_VENDOR = bcm
+
+VENDOR_REV ?= 0
+
+# Translate hardware revision to ONIE hardware revision
+ifeq ($(VENDOR_REV),0)
+  MACHINE_REV = 0
+else
+  $(warning Unknown VENDOR_REV '$(VENDOR_REV)' for MACHINE '$(MACHINE)')
+  $(error Unknown VENDOR_REV)
+endif
+
+# The VENDOR_VERSION string is appended to the overal ONIE version
+# string.  HW vendors can use this to appended their own versioning
+# information to the base ONIE version string.
+VENDOR_VERSION = .0.3
+
+# Vendor ID -- IANA Private Enterprise Number:
+# http://www.iana.org/assignments/enterprise-numbers
+# Lenovo 
+VENDOR_ID = 19046
+
+# Enable the i2ctools and the onie-syseeprom command for this platform
+I2CTOOLS_ENABLE = yes
+I2CTOOLS_SYSEEPROM = no
+
+# Console parameters
+CONSOLE_DEV = 1
+
+#Enable UEFI support
+UEFI_ENABLE = yes
+# Set Linux kernel version
+LINUX_VERSION		= 4.1
+LINUX_MINOR_VERSION	= 38
+
+# Specify uClibc version
+#UCLIBC_VERSION = 0.9.32.1
+GCC_VERSION = 4.9.2
+
+#Extra kernel command line
+EXTRA_CMDLINE_LINUX ?= quiet
```

Comment:
quiet is already added to the kernel command line

Reply:
I just have checked it and im still seeing when demo OS boots up it prints kernel msgs in console. So, I will keep this flag ON.

Reply:
The demo OS does not use `quiet`, you are right.  That was on purpose to see that things were working.

When ONIE boots, it uses `quiet`.

If you want demo OS to also be quiet, then this is fine.